lib/game_team.rb

```diff
@@ -1,8 +1,16 @@
 class GameTeam
-  attr_reader :game_id, :team_id, :hoa, :result, :head_coach, :goals, :shots, :tackles, :parent
+  attr_reader :game_id,
```

Comment:
When we get to the end of this project we should take another look at our attr_readers and delete any that we never actually call. Per Kat's advice in the library independent challenge review I think we want to keep all of the data as instance variables, but there is no sense in giving them reader methods if we are not going to use them. We could also consider storing the row as an instance variable in order to preserve that data, and then only breaking out distinct instance variables only for the data we need to read